Test/comp/Datatype.dfy.expect

```diff
@@ -18,7 +18,7 @@ Stream.Next
 {Berry.Smultron, Berry.Jordgubb, Berry.Hallon}
 CoBerry.Hjortron true true false
 false
-42 q hello
+42 'q' hello
```

Comment:
It's strange to me for this to be an artifact to switching to proper Unicode characters, but I'm not opposed to it.

Reply:
It's an intentional side-effect, although only documented internally: https://github.com/dafny-lang/dafny/pull/3629

As you're not the first to ask it might be worth following up on with documentation, but since the behavior of `print` is all but unspecified in the reference manual or anywhere else I'm not sure it's worth it. :)